stix_shifter_modules/verify/README.md

```diff
@@ -0,0 +1,63 @@
+# Verify
+
+This is a connector for searching IBM Verify events. 
```

Comment:
Some details about the API, query language and the data schema would be useful for the readme.